### PR TITLE
Bug fixes and volo related additions

### PR DESCRIPTION
**Pull request by T-Nicholls**

### CODE REVIEW

File: atip/sim_data_sources.py [hunk 1]
```diff
@@ -435,7 +443,7 @@ def get_value(self, field, handle=None, throw=True):
                 logging.warning("Potentially out of date data returned. " +
                                 error_msg)
         if field in list(self._field_funcs.keys()):
-            if field.startswith('phase'):
```
Comment (willrogers) [inline atip/sim_data_sources.py]: This could use a comment.

**Comment by aawdls** [comment]
Currently trying to get set up to test the virtac off this branch.

The install instructions need updating for Pipenv, and the URL in INSTALL.rst needs updating to dls-controls from T-Nicholls

File: atip/utils.py [hunk 2]
```diff
@@ -94,80 +95,90 @@ class elems():
...
+                                                        from which an
+                                                        ATSimulator object can
+                                                        be extracted.
 
     Returns:
         cothread.Thread: The calculation thread that the Pytac lattice's
                          ATSimulator object uses to recalculate physics data.
     """
-    return get_atsim(pytac_lattice)._calculation_thread
+    return get_atsim(target)._calculation_thread
```
Comment (willrogers) [inline atip/utils.py]: Hm this isn't public. Does it need to be?

**Comment by aawdls** [comment]
It looks like we built accelerator-toolbox against numpy 1.16, whereas it says that it requires numpy>=1.10. The numpy in DLS site_packages is 1.13 and is incompatible. You get an error on startup of the virtac. This can be fixed by installing numpy 1.16 into the Pipenv in the virtac directory. We could maybe add numpy 1.16 to the Pipfile here. Or maybe there's a better way.

**Comment by T-Nicholls** [comment]
@aawdls well caught; there is now an issue with a proposed fix here https://github.com/atcollab/at/issues/128, as the problem should really be fixed rather than patching around it at our end.

**Pull request merged**